src/20_Components/amp-access.html

```diff
@@ -21,8 +24,13 @@
   the authorization response -->
   <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
 
-  <!-- Add the configuration for the `authorization`, `pingback` and the `login` endpoints. We provide two login endpoints, one for sign-in and one for sign-out. It is
-  also possible to create a fallback response that will be used if the authorization fails with the `authorizationFallbackResponse` attribute -->
+  <!-- Add the configuration for the `authorization` and the `login` endpoints. We provide two login endpoints, one for sign-in and one for sign-out. It is
+    also possible to create a fallback response that will be used if the authorization fails with the `authorizationFallbackResponse` attribute.
+
+    The [Pingback Endpoint](https://www.ampproject.org/docs/reference/components/amp-access#pingback-endpoint) is also defined,
```

Comment:
Suggestion, rephrase to: "The [Pingback Endpoint](https://www.ampproject.org/docs/reference/components/amp-access#pingback-endpoint) will be called when a user views the document, which can be used to update metering information.